Sources/DesignSystem/Views/BraveButtonStyle.swift

```diff
@@ -31,6 +31,7 @@ public struct BraveButtonSize {
 public struct BraveFilledButtonStyle: ButtonStyle {
   @Environment(\.isEnabled) private var isEnabled
 
+  private let clipShape = RoundedRectangle(cornerRadius: 48, style: .continuous)
```

Comment:
Need @kylehickinson for this line, to make sure we do not break this button in other views

Reply:
We do not want to use 48 instead of `Capsule`, the background shape will max out then. What's the goal with this change?

Reply:
Dynamic font support

<img width="294" alt="Zrzut ekranu 2022-10-3 o 18 32 56" src="https://user-images.githubusercontent.com/6950387/193633067-ac179075-23f7-4a51-a83e-c5a73e53d14a.png">